README.md

```diff
@@ -1,2 +1,53 @@
-# tic-tac-toe
-Tic Tac Toe game app project
+# TIC TAC TOE
+
+In this project we build a Tic Tac Toe game app with Ruby. The game is played in the Terminal.
+
+## Game Description
```

Comment:
- [ ] README file explicitly tells how to start the game.

Your game description is great, but you have your game description along with the instructions, you can separate them and add the instructions in a small list.